lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -967,7 +953,8 @@ class HTMLInputElementImpl extends HTMLElementImpl {
       // than the minimum, the element is simultaneously suffering from an underflow and suffering
       // from an overflow.
       const reversedRangeSufferingOverUnderflow = () => {
-        return this._parsedValueIsNumber && this._parsedValue > this._maximum && this._parsedValue < this._minimum;
+        const parsedValue = this._convertStringToNumber(this._value);
+        return typeof parsedValue === "number" && parsedValue > this._maximum && parsedValue < this._minimum;
```

Comment:
Can we replace the typeof === number with !== null?

Reply:
Done.